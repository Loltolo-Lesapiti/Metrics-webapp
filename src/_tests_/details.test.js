import renderer from "react-test-renderer";
import Details from "./mock/details";

describe("Currency details", () => {
  it("Check if it renders correctly", () => {
    const tree = renderer.create(<Details />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
