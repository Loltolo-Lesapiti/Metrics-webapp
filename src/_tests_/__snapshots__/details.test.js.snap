// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Currency details Check if it renders correctly 1`] = `
Array [
  <h3>
    Currency Displays
  </h3>,
  <div
    className="list-group"
    onKeyDown={[Function]}
  >
    <div
      className="list-group-item"
      data-rr-ui-event-key={null}
      onClick={[Function]}
    >
      <table
        className="table table-sm table-striped table-bordered table-hover"
      >
        <thead>
          <tr>
            <th>
              #
            </th>
            <th>
              Currency Details
            </th>
            <th>
              Figures as it stands
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              1
            </td>
            <td>
              Ticker
            </td>
            <td>
              EUR/USD
            </td>
          </tr>
          <tr>
            <td>
              2
            </td>
            <td>
              Bid
            </td>
            <td>
              0.98384
            </td>
          </tr>
          <tr>
            <td>
              3
            </td>
            <td>
              Ask
            </td>
            <td>
              0.98384
            </td>
          </tr>
          <tr>
            <td>
              4
            </td>
            <td>
              Open
            </td>
            <td>
              0.97861
            </td>
          </tr>
          <tr>
            <td>
              5
            </td>
            <td>
              Low
            </td>
            <td>
              0.97049
            </td>
          </tr>
          <tr>
            <td>
              6
            </td>
            <td>
              High
            </td>
            <td>
              0.98680
            </td>
          </tr>
          <tr>
            <td>
              7
            </td>
            <td>
              Change
            </td>
            <td />
          </tr>
          <tr>
            <td>
              8
            </td>
            <td>
              Date
            </td>
            <td>
              2022-10-21 14:33:05
            </td>
          </tr>
        </tbody>
      </table>
    </div>
    <div
      className="list-group-item"
      data-rr-ui-event-key={null}
      onClick={[Function]}
    >
      <table
        className="table table-sm table-striped table-bordered table-hover"
      >
        <thead>
          <tr>
            <th>
              #
            </th>
            <th>
              Currency Details
            </th>
            <th>
              Figures as it stands
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              1
            </td>
            <td>
              Ticker
            </td>
            <td>
              USD/JPY
            </td>
          </tr>
          <tr>
            <td>
              2
            </td>
            <td>
              Bid
            </td>
            <td>
              148.246
            </td>
          </tr>
          <tr>
            <td>
              3
            </td>
            <td>
              Ask
            </td>
            <td>
              148.246
            </td>
          </tr>
          <tr>
            <td>
              4
            </td>
            <td>
              Open
            </td>
            <td>
              150.152
            </td>
          </tr>
          <tr>
            <td>
              5
            </td>
            <td>
              Low
            </td>
            <td>
              146.183
            </td>
          </tr>
          <tr>
            <td>
              6
            </td>
            <td>
              High
            </td>
            <td>
              151.947
            </td>
          </tr>
          <tr>
            <td>
              7
            </td>
            <td>
              Change
            </td>
            <td />
          </tr>
          <tr>
            <td>
              8
            </td>
            <td>
              Date
            </td>
            <td>
              2022-10-21 14:33:05
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>,
]
`;
